src/game/logic/object/update/physicsupdate.h

```diff
@@ -26,20 +28,29 @@ enum PhysicsTurningType
     TURN_POSITIVE,
 };
 
+// This needed to be moved out of the class in order for DEFINE_ENUMERATION_BITWISE_OPERATORS to work
+enum PhysicsFlagsType
+{
+    STICK_TO_GROUND = 1,
+    ALLOW_BOUNCE = 2,
+    APPLY_FRICTION2D_WHEN_AIRBORNE = 4,
+    UPDATE_EVER_RUN = 8,
+    WAS_AIRBORNE_LAST_FRAME = 16,
+    ALLOW_COLLIDE_FORCE = 32,
+    IS_AIRBORNE = 64,
+    MOVING = 128,
+    FLAG_UNK3 = 256,
+    FREEFALL = 512,
+    UPDATING = 1024,
+    STUNNED = 2048,

```

Comment:
These enum value names are now in global namespace. To avoid name collisions, can either prefix them with `PHYSICS_` or use `enum class`, if possible.

Reply:
Came up with a better solution.